test.js

```diff
@@ -137,3 +137,19 @@ describe('runAsync', function () {
     }, /No Promise Implementation/);
   });
 });
+
+describe('runAsync.cb', function () {
+  it('handles callback parameter', function (done) {
```

Comment:
it was a good idea, the new implementation of `runAsync.cb` wasn't returning the value.